Makefile

```diff
@@ -1,6 +1,5 @@
 NAME := wormbase/datomic-to-catalyst
 VERSION ?= $(shell git describe --abbrev=0 --tags)
-WS_VERSION := WS257
 LOWER_WS_VERSION = `echo $(WS_VERSION) | tr A-Z a-z`
```

Comment:
I am not sure what this line is for, and there appears to be WS_VERSION in this file a couple more places. Any ideas?